scripts/rejected-payloads.py

```diff
@@ -98,14 +99,19 @@ def categorizeSingle(session, tag):
             print("%10d: %20s - %s" % (idx+1, reason, reject_reasons[reason]))
 
         while True:
-            val = input("Enter your selection between 1 and " + str(len(reject_reasons_keys)) + ": ")
+            val = input("Enter one of more selections between 1 and " + str(len(reject_reasons_keys)) + " separated by spaces: ")
             try:
-                index = int(val)
-                if index > 0 and index <= len(reject_reasons_keys):
+                # Values outside the legal range are ignored.  Invalid values throw an exception so you can
+                # try again.
+                selected_indexes = [i for i in map(int, val.split()) if 1 <= i <= len(reject_reasons_keys)]
+                if selected_indexes:
                     break
             except ValueError:
+                print(f"  This contains invalid integer values: {val}")
```

Comment:
"f strings" is a python3 feature -- if we want python2, I can change this to use the  percent method.